spec/examples/usual/dynamic_options/consists_of/example5_spec.rb

```diff
@@ -87,7 +87,7 @@
             expect { perform }.to(
               raise_error(
                 ApplicationService::Exceptions::Input,
-                "[Usual::Collection::Example5] Required element in input collection `ids` is missing"
+                "Input `ids` must be an array of `String`"
```

Comment:
This change is correct. Technically the attribute value is present. The fact that there is an incorrect value inside the collection is the responsibility of the dynamic option.